### PR TITLE
gr-modtool: Update the validation in Modtool Add

### PR DESCRIPTION
**Pull request by swap-nil7**
The validation of the arguments list and copyright holder should be updated to take care of the potential whitespaces. 
The check on the arguments list is currently incorrect.

### CODE REVIEW

File: gr-utils/python/modtool/cli/add.py [hunk 1]
```diff
@@ -101,15 +101,15 @@ def get_lang(self):
 
 def get_blockname(self):
     """ Get the blockname"""
-    if self.info['blockname'] is None:
+    if not self.info['blockname'] or self.info['blockname'].isspace():
         while not self.info['blockname'] or self.info['blockname'].isspace():
             self.info['blockname'] = cli_input("Enter name of block/code (without module name prefix): ")
     if not re.match('[a-zA-Z0-9_]+', self.info['blockname']):
```
Comment (mormj) [inline gr-utils/python/modtool/cli/add.py]: Strangely, I'm allowed to create a block with a space in the name (separate issue than this PR), but probably could be prevented here

Comment (swap-nil7) [reply gr-utils/python/modtool/cli/add.py]: @mormj: Yes, that's right. What do you think would be better, replacing spaces with an underscore or disallowing the users to use a space in the `blockname`?

Comment (mormj) [reply gr-utils/python/modtool/cli/add.py]: Personally I think you could go either way - it's a very fringe case that someone would have to intentionally do with quotes.  Just disallowing is probably the safest.

Comment (primercuervo) [reply gr-utils/python/modtool/cli/add.py]: I think avoiding spaces is the way to go. I really don't see a real use-case where I would like to manage spaces, needless to say that they are handled different for different OS. I suggest just disallowing spaces

File: gr-utils/python/modtool/cli/add.py [hunk 2]
```diff
@@ -101,15 +101,15 @@ def get_lang(self):
...
         while not self.info['blockname'] or self.info['blockname'].isspace():
             self.info['blockname'] = cli_input("Enter name of block/code (without module name prefix): ")
-    if not re.match('[a-zA-Z0-9_]+', self.info['blockname']):
+    if not re.match('^[a-zA-Z0-9_]*$', self.info['blockname']):
         raise ModToolException('Invalid block name.')
 
 def get_copyrightholder(self):
     """ Get the copyrightholder of the block to be added """
-    if self.info['copyrightholder'] is None:
+    if not self.info['copyrightholder'] or self.info['copyrightholder'].isspace():
```
[the remark this answers is not in the document]
Comment (swap-nil7) [reply gr-utils/python/modtool/cli/add.py]: Yes, currently if the copyrightholder is empty, then by default gr-blockname owner. We can make that mandatory if that is good to have. :)

**Pull request merged**